au3/modules/import-export/mod-mp3/ExportMP3.cpp

```diff
@@ -1367,18 +1198,20 @@ bool MP3Exporter::PutInfoTag(wxFFile& f, wxFileOffset off)
 
 wxString MP3Exporter::GetLibraryPath()
 {
-    wxRegKey reg(wxT("HKEY_LOCAL_MACHINE\\Software\\Lame for Audacity"));
-    wxString path;
-
-    if (reg.Exists()) {
-        wxLogMessage(wxT("LAME registry key exists."));
-        reg.QueryValue(wxT("InstallPath"), path);
-    } else {
-        wxLogMessage(wxT("LAME registry key does not exist."));
+    HKEY hKey;
+    const char* subkey = "Software\\Lame for Audacity";
+    const char* valueName = "InstallPath";
+    char pathBuffer[MAX_PATH];
+    DWORD pathBufferSize = sizeof(pathBuffer);
+    std::string path;
+
+    if (RegOpenKeyExA(HKEY_LOCAL_MACHINE, subkey, 0, KEY_READ, &hKey) == ERROR_SUCCESS) {
```

Comment:
Not sure about that, would xxxA function work well with non-ASCII paths?

Reply:
This is legacy code from back when mp3 was patented. It can be safely removed I think; we just need to use the binary